vesta/exceptions/command_exceptions.py

```diff
@@ -0,0 +1,14 @@
+class CommandRuntimeError(Exception):
```

Comment:
Should look a bit later the use of this

Reply:
If you're only using it in one file, maybe try setting it up in it, with another name. The problem here is, it is really a weird error name, it is not precise at all, and can't explain anything about the error. It's just an error to have an error. Bad usage, maybe you can do real errors inheriting of this kind of error.